### DISCUSSION TITLE
repeated workspace numbers on each monitor

### DISCUSSION THREAD

**Discussion by victorballester7** [Jas-SinghFSU/HyprPanel#511] [General]
Hi, 
I was wondering whether it is possible (in the case when `hide unoccupied` is on) to only show the labels concerning the workspace on that monitor. For example if I have launched programs on workspaces 1 (on monitor A), 2 (on monitor A), 7 (on monitor B) and 9 (on monitor A), I would like to see:
 - On monitor A:  1  2  9
 - On monitor B:  7
Instead of: 1  2  7  9 on both monitors.
Thanks!

**Comment by Jas-SinghFSU** [comment]
There is an issue open for this and I will look into it. Currently the bar relies on having workspace rules defined which determines what workspace shows in your module - absence of which shows the workspace on all monitors.

We should be able to handle this without workspace rules as well, so I'll see what's possible.
